Source/Shared/Storage/Storage.swift

```diff
@@ -38,28 +38,28 @@ public class Storage {
   }
 
   /// Used for async operations
-  public lazy var async: AsyncStorageAware = AsyncStorage(storage: self.interalStorage, serialQueue: self.serialQueue)
+  public lazy var async: AsyncStorage = AsyncStorage(storage: self.interalStorage, serialQueue: self.serialQueue)
 }
 
 extension Storage: StorageAware {
   public func entry<T: Codable>(ofType type: T.Type, forKey key: String) throws -> Entry<T> {
-    return try sync.entry(ofType: type, forKey: key)
+    return try self.sync.entry(ofType: type, forKey: key)
```

Comment:
Why do you need `self`? 🤔

Reply:
There 's `sync` function from `DispatchQueue` or from `Swift` 🤔 but Xcode complains